cg/store/api/findbusinessdata.py

```diff
@@ -232,31 +244,34 @@ def get_flow_cell(self, flow_cell_id: str) -> Flowcell:
             flow_cells=self.Flowcell.query, flow_cell_id=flow_cell_id, function="flow_cell_has_id"
         )
 
-    def flowcells(
-        self, *, status: str = None, family: models.Family = None, enquiry: str = None
-    ) -> Query:
+    def flowcells(self, *, status: str = None, family: Family = None, enquiry: str = None) -> Query:

```

Comment:
```suggestion
    def flow_cells(self, *, status: str = None, family: Family = None, enquiry: str = None) -> Query:

```

Reply:
I agree with your suggested changes, but I do not want to extend the scope of this already large PR further with this refactoring. It would be best to do in a separate PR.